breed/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
+    <link rel="stylesheet" href="../assets/vendor/simplecss/simple.min.css" />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/contact">Contact us</a>
+        <a href="/about">About us</a>
+        <a href="/breed" class="current">Breed Info</a>
+        <a href="/current">Current Litter</a>
+      </nav>
+    </header>
+    <main>
+      <h1>AussieDoodle Breed Info</h1>
+
+      <p>
+        The Aussiedoodle is a delightful hybrid breed created by crossing a
```

Comment:
Decide if its `AussieDoodle`, `Aussiedoodle`, or `Aussie Doodle`. I suspect the last one is actually the correct form.

Reply:
There are other places where the name is used, so update it across the board when you decide.